### PR TITLE
🏗Switch to .prettierrc to avoid breaking IDE extensions

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Many IDE extensions use `.prettierrc` as the source of truth for code formatting, now that we're using Prettier we can use this configuration file instead of inlining inside the `.eslintrc` config.

See: https://github.com/prettier/eslint-plugin-prettier#options

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "bracketSpacing": false,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "quoteProps": "preserve"
+}
```
Comment (rsimha) [inline .prettierrc]: Add a trailing newline character.

File: .eslintrc [hunk 2]
```diff
@@ -129,12 +129,7 @@
     ],
     "object-shorthand": [2, "properties", { "avoidQuotes": true }],
     "prefer-const": 2,
-    "prettier/prettier": [2, {
-      "bracketSpacing": false,
-      "singleQuote": true,
-      "trailingComma": "es5",
-      "quoteProps": "preserve"
-    }],
+    "prettier/prettier": [2],
```
Comment (rsimha) [inline .eslintrc]: I think you can do away with the square brackets around `2`.

File: .prettierrc (hunk 1 above)
Comment (kristoferbaxter) [reply .prettierrc]: Makes, sense, addressed.

File: .eslintrc (hunk 2 above)
Comment (kristoferbaxter) [reply .eslintrc]: Good call.

**Comment by kristoferbaxter** [comment]
Verified the change continued to format code as expected.

**Pull request merged**